server/channels/app/platform/web_hub.go

```diff
@@ -492,14 +495,19 @@ func (h *Hub) Start() {
 				if metrics := h.platform.metricsIFace; metrics != nil {
 					metrics.DecrementWebSocketBroadcastBufferSize(strconv.Itoa(h.connectionIndex), 1)
 				}
+
+				// Remove the broadcast hook information before precomputing the JSON so that those aren't included in it
+				msg, broadcastHooks, broadcastHookArgs := msg.RemoveBroadcastHooks()
```

Comment:
Looks like we'd be calling this and making a copy for each message, regardless of whether the hooks are set? If so it may have a performance impact given this is a particularly hot path.

Reply:
Great catch. I was specifically trying to be careful about making new copies in this code. I'll fix that